module/entities/helper.js

```diff
@@ -543,12 +543,40 @@ module.exports = class EntitiesHelper {
   }
 
    /**
-     * List roles by entity type.
-     * @method
-     * @name subEntitiesRoles
-     * @param entityId - entity id.
-     * @returns {Object} List of roles by entity id.
-    */
+   * List of Entities
+   * @method
+   * @name list
+   * @param bodyData - Body data.
+   * @returns {Array} List of Entities.
+   */
+  
+  static listByEntityIds( bodyData ) {
```

Comment:
@aman-tunerlabs - Bodydata doesn't make sense if other people want to call this function. Please divide into different logical parameters